### PR TITLE
Support nethermind plugin as data source

### PR DESCRIPTION
**Pull request by jaensen**

### CODE REVIEW

File: CirclesUBI.PathfinderUpdater.IndexerDb/Block.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@ namespace CirclesUBI.PathfinderUpdater;
 
 public static class Block
 {
-    public static async Task<long> FindByTransactionHash(string connectionString, string transactionHash, Queries queries)
+    public static async Task<long> FindLatestBlockNumber(string connectionString, Queries queries)
```
Comment (benjaminbollen) [inline CirclesUBI.PathfinderUpdater.IndexerDb/Block.cs:7]: just curious how the function name and signature changed but nothing else in the function?

File: CirclesUBI.PathfinderUpdater.IndexerDb/Queries.cs [hunk 2]
```diff
@@ -70,45 +70,46 @@ from cache_crc_balances_by_safe_and_token
     order by ""blockNumber"" desc, ""transactionIndex"" desc, ""logIndex"" desc;
     ";
 
+    // TODO: Use the Discounted event from 0.3.4 to get precise demurraged balances
```
Comment (benjaminbollen) [inline CirclesUBI.PathfinderUpdater.IndexerDb/Queries.cs:73]: should we already store "when" (on what day) we've queried these balances?

**Review by benjaminbollen — APPROVED** [review]
Mostly FIPF, a little bit LGTM - because im very new to this code

File: CirclesUBI.PathfinderUpdater.IndexerDb/Block.cs (hunk 1 above)
Comment (benjaminbollen) [reply CirclesUBI.PathfinderUpdater.IndexerDb/Block.cs:7]: I think i understand how it worked before, and see that now you want to go over every block?

Comment (jaensen) [reply CirclesUBI.PathfinderUpdater.IndexerDb/Block.cs:7]: Good catch. That's basically dead code that was once needed to do partial updates to the pathfinder.

**Pull request merged**